tests/para-tests-no-ci/test-xcm-para.ts

```diff
@@ -8,7 +8,7 @@ import { createBlockWithExtrinsicParachain, logEvents, waitOneBlock } from "../u
 import { KeyringPair } from "@polkadot/keyring/types";
 import { ApiPromise } from "@polkadot/api";
 
-const palletId = "0x6D6f646c617373746d6E67720000000000000000";
+const palletId = "0x6d6f646c617373746d6e67720000000000000000";
```

Comment:
What is the type of palletId in rust?

Reply:
Ideally we would like the api to return that checksummed dont we

Reply:
yeah but it was failing for whatever reason in the comparison. The pallet is returning a non-checksummed address. Maybe it has to do with not having the accountId20 type there yet.

Reply:
I would prefer you keep the checksummed address and use toLowercase() that way we can easily find it bacl later

Reply:
Sounds good!